src/UI/StaffCraftingMenu.cpp

```diff
@@ -157,6 +158,7 @@ namespace UI
 
 			auto filterFlag = FilterFlag::None;
 			if (!disallowHeartStones && object == DLC2HeartStone) {
+				heartStoneCount = item->countDelta;
```

Comment:
I think this should be a `+=`.

Reply:
The `=` might be better, since you can't select multiple morpholiths - though there should only really be one heart stone type, so it probably doesn't make a difference

Reply:
I don't know if it can appear more than once in the loop. I suspect that, for example, if you have stolen heart stones, they might appear separately, but in which case we would still want to be able to use all of them for enchanting.